### PR TITLE
[4.0] Move route calls to plugin layouts

### PR DESCRIPTION
**Pull request by SharkyKZ**
Pull Request for Issue # .

### Summary of Changes

This just moves route calls to layouts in plugins.

### Testing Instructions

Code review or test that plugin work like before.

### Documentation Changes Required

🤷‍♂

### CODE REVIEW

**Comment by brianteeman** [comment]
For the less skilled among us - what is the benefit/purpose of this change

File: plugins/content/pagebreak/pagebreak.php [hunk 1]
```diff
@@ -384,7 +381,7 @@ protected function _createNavigation(&$row, $page, $n)
 
 		if ($page < $n - 1)
 		{
-			$links['next'] = Route::_(ContentHelperRoute::getArticleRoute($row->slug, $row->catid, $row->language) . '&limitstart=' . ($page + 1));
+			$links['next'] = ContentHelperRoute::getArticleRoute($row->slug, $row->catid, $row->language) . '&limitstart=' . ($page + 1);
```
Comment (infograf768) [inline plugins/content/pagebreak/pagebreak.php]: Would it be better if we use
`use Joomla\Component\Content\Site\Helper\RouteHelper;`

and therefore
`$links['next'] = RouteHelper::getArticleRoute($row->slug, $row->catid, $row->language) . '&limitstart=' . ($page + 1);`

What do you think?

**Comment by SharkyKZ** [comment]
@brianteeman It's to allow simple manipulation of the URL. Once the URL is passed through `Route::_()` manipulating it becomes more difficult.

File: plugins/content/pagebreak/pagebreak.php (hunk 1 above)
Comment (SharkyKZ) [reply plugins/content/pagebreak/pagebreak.php]: Yes, we should totally start using real classes at some point. But not sure whether to include this change in this PR.

Comment (infograf768) [reply plugins/content/pagebreak/pagebreak.php]: Maybe do it in a single PR later on? @wilsonge ?

Comment (SharkyKZ) [reply plugins/content/pagebreak/pagebreak.php]: Sure, I can do that later. Or you can do if you want.

Comment (wilsonge) [reply plugins/content/pagebreak/pagebreak.php]: Yeah single PR later on

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**